apps/dashboard/config/configuration_singleton.rb

```diff
@@ -42,6 +42,7 @@ def boolean_configs
       :bc_dynamic_js                => false,
       :bc_simple_auto_accounts      => false,
       :bc_clean_old_dirs            => false,
+      :bc_show_fields_prefill       => false,
```

Comment:
I'm not sure if we want a configuration for this. I mean - I don't really see any use in turning it off, or at least cannot imagine the reason to.

Reply:
Removed